jwst/photom/photom.py

```diff
@@ -34,52 +36,53 @@ def __init__(self, model):
             input Data Model object
 
         """
-        self.input = model
+        # Create a copy of the input model
+        self.input = model.copy()
 
         self.instrument = model.meta.instrument.name.upper()
         self.detector = model.meta.instrument.detector.upper()
         self.exptype = model.meta.exposure.type.upper()
         self.filter = None
-        if model.meta.instrument.filter is not None:
+        if model.meta.instrument.filter:
```

Comment:
Why this change? This implies `filter` can be `False` .

Reply:
It's intended to just check for whether ``filter`` is populated with anything in the model, because some instrument modes don't use the filter keyword. I was recently told this was a more "pythonic" way to check for its existence, but I could easily be wrong.

Reply:
If it's not populated then it's `None`. The way to check for `None` is to use `is None`

Reply:
OK, I've put back all the instances of ``is not None`` that were there originally.